packages/ui/src/Editor.tsx

```diff
@@ -20,7 +21,12 @@ export function Editor() {
           end={<Viewport />}
         />
       }
-      end={<Timeline />}
+      end={
+        <div>
```

Comment:
I think some CSS will be needed here because right now the timeline doesn't expand correctly:
![image](https://user-images.githubusercontent.com/64662184/223293036-ae195e74-54fc-4039-8aa2-6d6792de8121.png)